### PR TITLE
Split Issue And Repo Handling Into Separate Workflows

### PR DESCRIPTION
**Pull request by m4s-b3n**
successfull runs of dispatched workflows:

Creation: https://github.com/xpirit-training/demo-features-issueops/actions/runs/5890115794
Deletion: https://github.com/xpirit-training/demo-features-issueops/actions/runs/5890126934

Toplevel WF can be tested after merge as this requires issue handling which is done in main branch

### CODE REVIEW

**Comment by aatmmr** [comment]
@spindev and @m4s-b3n - is this now related to #48?

File: .github/workflows/notify-issue-handling.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Notify About Started Workflow
+
+on:
+  workflow_call:
```
Comment (aatmmr) [inline .github/workflows/notify-issue-handling.yml:4]: Add optional message to be passed in?

File: .github/workflows/notify-issue-handling.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+jobs:
+  notify:
+    name: "Parse Deletion Issue"
```
Comment (aatmmr) [inline .github/workflows/notify-issue-handling.yml:8]: Name doesn't fit?

**Review by aatmmr — CHANGES_REQUESTED** [review]
I think the notify-issue-handling workflow should be an action. I think a composite action would be sufficient.

File: .github/workflows/notify-issue-handling.yml (hunk 2 above)
Comment (m4s-b3n) [reply .github/workflows/notify-issue-handling.yml:8]: yepp, thx

File: .github/workflows/notify-issue-handling.yml (hunk 1 above)
Comment (m4s-b3n) [reply .github/workflows/notify-issue-handling.yml:4]: I'll do when we need it 🤞

File: .github/workflows/handle-issue.yml [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+  handle-request:
+    needs:
+      - parse-request
+    uses: ./.github/workflows/repo-request.yml
+    with:
+      project_name: ${{ needs.parse-request.outputs.project_name }}
+      repo_name: ${{ needs.parse-request.outputs.repo_name }}
+      visibility: ${{ needs.parse-request.outputs.visibility }}
+      language: ${{ needs.parse-request.outputs.language }}
+    if: ${{ github.event.label.name != 'repo-request' }}
```
[the remark this answers is not in the document]
Comment (m4s-b3n) [reply .github/workflows/handle-issue.yml]: kay :)

[the remark this answers is not in the document]
Comment (m4s-b3n) [reply .github/workflows/handle-issue.yml]: done

File: .github/workflows/notify-issue-handling.yml (hunk 2 above)
Comment (m4s-b3n) [reply .github/workflows/notify-issue-handling.yml]: done

**Comment by m4s-b3n** [comment]
> Awesome work @m4s-b3n 🚀
> 
> I think the notify-issue-handling workflow should be an action. I think a composite action would be sufficient.

done

**Comment by m4s-b3n** [comment]
@aatmmr @spindev pls have a closer look at the composite actions and how they are used as these are my first ones 😉

**Review by spindev — APPROVED** [review]
LGTM. nice one!

Actions are good =)

**Comment by aatmmr** [comment]
@m4s-b3n and @spindev We need to consider the job names

image

As a user I do not know what I am supposed to review.

**Pull request merged**